lineapy/plugins/jinja_templates/module.jinja

```diff
@@ -10,4 +14,15 @@ def run_all_sessions({{module_input_parameters}}):
 {{indentation_block}}return artifacts
 
 if __name__ == "__main__":
-{{indentation_block}}run_all_sessions()
\ No newline at end of file
+{{indentation_block}}# Edit this section to customize the behavior of artifacts
+{% if module_input_parameters != '' -%}
+{{indentation_block}}parser = argparse.ArgumentParser()
+{% for parser_block in parser_blocks -%}
+{{indentation_block}}{{parser_block}}
+{% endfor -%}
+{{indentation_block}}args = parser.parse_args()
+{{indentation_block}}artifacts = run_all_sessions({{parser_input_parameters}})
+{% else -%}
+{{indentation_block}}artifacts = run_all_sessions()
+{% endif -%}
+{{indentation_block}}print(artifacts)
```

Comment:
are we planning on printing at the end of pipelines or was this for debugging? Not sure if such prints would change output logs and cause downstream issues for users. leaving it to @mingjerli for final say.

Reply:
I've been using `subprocess.run` to execute the python module itself and capture the print result to compare the expected value. 

https://github.com/LineaLabs/lineapy/blob/33d27209ae617e01754107fa77a12cad6e4bfa62/tests/unit/graph_reader/test_artifact_collection.py#L227-L231

Like the comment in the template, we expect users to edit this section to customize how they want to deal with calculated artifacts.

The other two options are doing nothing or saving somewhere and I don't think we will have a destination that satisfies every user.